mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -964,6 +964,20 @@ private[python] class PythonMLLibAPI extends Serializable {
       points.asScala.toArray)
   }
 
+  /**
+   * Java stub for the update method of StreamingKMeansModel.
+   */
+  def updateStreamingKMeansModel(
+        clusterCenters: java.util.ArrayList[Vector],
```

Comment:
4-space indentation

`java.util.List` should be sufficient, which is already aliased as `JList`